src/domains/pay-wallet-app/api/io-payment-wallet/v1/_get_payment_methods.xml.tpl

```diff
@@ -10,6 +10,35 @@
     </inbound>
     <outbound>
       <base />
+      <choose>
+        <when condition="@("true".Equals("{{enable-pm-ecommerce-io}}"))">
```

Comment:
TODO: add here the family&friends check as per https://github.com/pagopa/pagopa-infra/pull/2168/files

Reply:
Fixed in 5a7b76c3